OpenDreamRuntime/Objects/Types/DreamObjectSavefile.cs

```diff
@@ -118,15 +133,14 @@ public sealed class SavefileDirectory : Dictionary<string, DreamValue> { }
                 return true;
             case "dir":
                 DreamList dirList = ObjectTree.CreateList();
-
-                foreach (string dirPath in Directories.Keys) {
-                    if (dirPath.StartsWith(_currentDirPath)) {
-                        dirList.AddValue(new DreamValue(dirPath));
-                    }
-                }
+                // TODO reimplement
```

Comment:
loop CurrentDir keys, go deeper if children vals have keys

Reply:
Resolved - afaict BYOND just dumps the directories in the current working directory, not subdirs. I have not implemented the save-backed list that would be needed for remove + add to work because I am so done. Added it as a TODO for later.